index.html

```diff
@@ -145,43 +145,51 @@ <h3 class="project-heading">Multi-Post Stories</h3>
           <li>html</li>
           <li>css</li>
           <li>javascript</li>
+          <li class="hidden">Ruby on rails</li>
         </ul>
         <button type="button">See Project</button>
       </section>
     </div>
     <footer id="footer">
       <section id="about-myself">
-        <h2 id="about-myself-heading">About Myself</h2>
-        <p id="about-myself-description">
-          Hello I'm a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don't hestiate to
-          contact me.
-        </p>
-        <section class="contacts">
-          <h2 class="contacts-heading">LET'S CONNECT</h2>
-          <ul class="social-media-icons no-bullets">
-            <li>
-              <a href="https://github.com/NayanKaran">
-                <img alt="github icon" src="./images/icons/github_icon.svg" />
-              </a>
-            </li>
-            <li>
-              <a href="https://www.linkedin.com/in/debabrata-mandal-83461696/">
-                <img
-                  alt="linkedin icon"
-                  src="./images/icons/linkedin_icon.svg"
-                />
-              </a>
-            </li>
-            <li>
-              <a href="https://twitter.com/bapidebabrata">
-                <img alt="twitter icon" src="./images/icons/twitter_icon.svg" />
-              </a>
-            </li>
-          </ul>
-        </section>
-        <button type="button">Get my resume</button>
+        <div>
+          <h2 id="about-myself-heading">About Myself</h2>
+          <p id="about-myself-description">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <section class="contacts">
+            <h2 class="contacts-heading">LET'S CONNECT</h2>
+            <ul class="social-media-icons no-bullets">
+              <li>
+                <a href="https://github.com/NayanKaran">
+                  <img alt="github icon" src="./images/icons/github_icon.svg" />
+                </a>
+              </li>
+              <li>
```

Comment:
- [ ] Please kindly ensure that the spacing at the top of the headline section matches that of the design. At the moment, your spacing is more than what was specified in the original design.
- [ ] Please also ensure that the background image is properly set as in the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/165749414-4d7edc9d-e936-4f40-989d-3091738a79d8.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/165749532-3d2760f5-bd1c-4b71-9ed4-0077e4aea35f.png)

Reply:
Hi @AnselemOdims,  I checked the margin and padding values they are as specified by the design. can you please clarify this?

Thanks for pointing out the background image. I am going to change it.